.goreleaser.yaml

```diff
@@ -0,0 +1,129 @@
+# run a local "snapshot" build without publishing:
+#    TRAVIS_BUILD_NUMBER=1 goreleaser --snapshot --rm-dist --debug
+#
+project_name: chezmoi
+
+builds:
+- binary: chezmoi
+  env:
+    - CGO_ENABLED=0
+  ldflags:
+     - -s -w
+  goos:
```

Comment:
I went a bit overboard with the OS's and architectures, but goreleaser makes it so easy it is hard not to resist.

I selfishly request these platforms because it's where I intend to run chezmoi:
* linux/amd64
* openbsd/amd64
* macos